waffle_hub/dataset/dataset.py

```diff
@@ -621,6 +622,9 @@ def _import_object_detection(set_dir: Path, image_ids: list[int]):
             # import other field
             _import(set_dir, image_ids)
 
+        # TODO: add unlabeled set
+        io.save_json([], ds.unlabeled_set_file, create_directory=True)
```

Comment:
@siegkeron This is for you.
`unlabeled_set_file` must exist (software aspect).
So, I fixed #141 by adding this line.
You should run `from_xxx` again.

Reply:
or you just create `your/dataset/path/sets/unlabeled.json` with empty list to avoid re-run `from_xxx` function.

like below.
```
[]
```